java-hello-world/README.md

```diff
@@ -0,0 +1,8 @@
+# Java Hello World with GraalVM Enterprise
```

Comment:
We already have a Java hello world sample, contributed by Cris Bensen, https://github.com/graalvm/graalvm-demos/tree/master/hello-graal. This one is built with maven. Let's maybe use this clarification in the project name or extend this README?

Reply:
Renamed the sample to 'java-hello-world-maven'.